block/retriever.go

```diff
@@ -35,11 +37,14 @@ func (m *Manager) RetrieveLoop(ctx context.Context) (err error) {
 func (m *Manager) syncToTargetHeight(targetHeight uint64) error {
 	for currH := m.State.NextHeight(); currH <= targetHeight; currH = m.State.NextHeight() {
 		// if we have the block locally, we don't need to fetch it from the DA
-		err := m.processLocalBlock(currH)
+		err := m.applyLocalBlock(currH)
 		if err == nil {
 			m.logger.Info("Synced from local", "store height", currH, "target height", targetHeight)
 			continue
 		}
+		if !errors.Is(err, gerrc.ErrNotFound) {
+			m.logger.Error("Process local block", "err", err)
```

Comment:
```suggestion
			m.logger.Error("Apply local block", "err", err)
```
Since you renamed `processLocalBlock` to `applyLocalBlock`, I think it would make sense to log it as such

Reply:
done